unified-installer/astra-unified-installer.sh

```diff
@@ -12,6 +12,9 @@
 _PROBLEMS=() # Any failed checks will be added to this array and the program will exit at specific checkpoints if not empty
 _CONFIG_BUILDER=() # Contains the fully resolved config to be output at the end during a dry run
 _KUBERNETES_VERSION=""
+# _RESOURCES_OR_PATCHES_HAVE_BEEN_APPLIED is to determine whether the script did anything or not, so we can
+# let the user if it was a no-op.
+_RESOURCES_OR_PATCHES_HAVE_BEEN_APPLIED="false"
```

Comment:
Messaging wasn't very clear when the script ended up doing nothing. This will be used to output a message at the very end telling the user if the script was a noop.